src/components/group/appsco-company-group-resources.html

```diff
@@ -0,0 +1,511 @@
+                    value: function() {
+                        return [];
+                    }
+                },
+
+                _resourcesEmpty: {
+                    type: Boolean,
+                    value: false
+                },
+
+                _message: {
+                    type: String,
+                    value: ''
+                },
+
+                _nextPageApiUrl: {
+                    type: String
+                },
+
+                _totalResources: {
+                    type: Number,
+                    value: 0
+                },
+
+                _renderedIndex: {
+                    type: Number,
+                    value: -1
+                },
+
+                /**
+                 * Animation for loader appearance.
+                 */
+                animationConfig: {
+                    value: function() {
+                        return {
+                            'entry': {
+                                name: 'cascaded-animation',
+                                animation: 'slide-from-left-animation',
+                                nodes: [],
+                                nodeDelay: 50,
+                                timing: {
+                                    duration: 300
+                                }
+                            }
+                        }
+                    }
+                }
+            },
+
+            behaviors: [
+                Polymer.NeonAnimationRunnerBehavior
+            ],
+
+            _computeHeaders: function(authorizationToken) {
+                return {
+                    'Authorization': 'token ' + authorizationToken
+                }
+            },
+
+            _onListApiChanged: function(url) {
+
+                if (url && url.length > 0) {
+                    this._loadResources();
+                }
+            },
+
+            _setLoadMoreAction: function() {
+                this._loadMore = (this.loadMore && this._allResources.length < this._totalResources);
+            },
+
+            _hideLoadMoreAction: function() {
+                this._loadMore = false;
+            },
+
+            _loadResources: function() {
+                this._showProgressBar();
+                this._loadMore = false;
+                this.set('_resources', []);
+                this.set('_allResources', []);
+                this.$.getGroupResourcesApiRequest.generateRequest();
+            },
+
+            reloadResources: function() {
+                this._loadResources();
+            },
+
+            _loadMoreResources: function() {
+                this._showLoadMoreProgressBar();
+                this.$.getGroupResourcesApiRequest.url = this._nextPageApiUrl;
+                this.$.getGroupResourcesApiRequest.generateRequest();
+            },
+
+            _onError: function() {
+                this._message = 'We couldn\'t load accounts at the moment. Please try again in a minute.';
```

Comment:
Instead of 'We couldn\'t load accounts at the moment. Please try again in a minute.' should be 'We couldn\'t load resources at the moment. Please try again in a minute.'